tests/eleventy-site/_site/_redirects

```diff
@@ -0,0 +1,4 @@
+/force              /test                           200!
```

Comment:
I believe these files are redundant? They were moved under `tests/integration` in https://github.com/netlify/cli/pull/4164